docs/src/design/story/bundles/training/EXTENSIONS.md

```
@@ -0,0 +1,532 @@
+      target: 10,
+      against: { piece_id: "player", property: "charm" },
+      modifier: 0,
+      success_text: "12/20 chance"
+    }
+  }
+}
+```
+
+CLI rendering:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)